optuna/storages/_grpc/api.proto

```diff
@@ -378,17 +378,18 @@ message GetTrialReply {
 }
 
 /**
- * Request to get all trials in a study.
+ * Request to get trials in a study.
  */
-message GetAllTrialsRequest {
+message GetTrialsRequest {
     int64 study_id = 1;
-    repeated TrialState states = 2;
+    repeated int64 included_trial_ids = 2;
```

Comment:
Probably, `including` is better if I understand the context of this request.
```suggestion
    repeated int64 including_trial_ids = 2;
```

Reply:
I think the current naming and implementation are consistent with the `RDBStorage` ([ref](https://github.com/optuna/optuna/blob/master/optuna/storages/_rdb/storage.py#L812)).